MATLAB/Utilities/checkGeo.m

```diff
@@ -72,7 +72,7 @@
 
 if isfield(geo,'offOrigin')
     assert(isequal(size(geo.offOrigin),[3 1]) | isequal(size(geo.offOrigin),[3 size(angles,2)]),'TIGRE:checkGeo:BadGeometry','geo.offOrigin Should be 3x1 or 3xsize(angles,2)')
-    assert(isa(geo.offOrigin,'double'),'TIGRE:checkGeo:BadGeometry','Field geo.offOrigin is not double type.' )
+    assert(isa(geo.offOrigin,'double'),'TIGRE:checkGeo:BadGeometry','Field geo.offOrigin is not double type.' )% "we could cast it to double" 

```

Comment:
(this is so you know how code-reviews work). 

This is the only line you changed but it does not address the issue #91. Of course, this can be just a test.